packages/create-sitecore-jss/src/common/prompts/base.ts

```diff
@@ -1,14 +1,23 @@
 import chalk from 'chalk';
-import { NextjsAnswer } from './NextjsAnswer';
-import { QuestionCollection } from 'inquirer';
+import { DistinctQuestion } from 'inquirer';
 
-export const userPrompts: QuestionCollection<NextjsAnswer> = [
-  // App name
+export enum FetchWith {
+  GraphQL = 'GraphQL',
+  REST = 'REST',
+}
+
+export interface ClientAppAnswer {
+  appName: string;
+  fetchWith: FetchWith;
+  hostName: string;
+}
+
+export const clientAppPrompts: DistinctQuestion[] = [
   {
     type: 'input',
     name: 'appName',
     message: 'What is the name of your app?',
-    default: 'sitecore-jss-nextjs',
+    default: 'sitecore-jss-app',
```

Comment:
Why change the default to app?
What template would this use?

Reply:
It's a base array of prompts, and we have different samples, we can't use `sitecore-jss-nextjs` name for RAV, etc.